native/cocos/scene/Camera.cpp

```diff
@@ -190,6 +190,9 @@ void Camera::update(bool forceUpdate /*false*/) {
 
     // view-projection
     if (viewProjDirty) {
+        Mat4 scaleMat{};
+        scaleMat.scale(_node->getWorldScale());
+        Mat4::multiply(scaleMat, _matView, &_matView);
```

Comment:
Have two questions:
- why need to remove scale matrix
- the logic seems to multiply scale matrix

Reply:
for this issue：https://github.com/cocos/3d-tasks/issues/10377

Reply:
Because _matView is an inverse worldMatrix, it needs to be corrected with the worldscale of node

Reply:
as shown below：

![image](https://user-images.githubusercontent.com/10766457/173477292-01121e96-c4b0-4d44-85a8-ecec9ea03df3.png)

Reply:
It will do reverse every frame. I think can cache the result. It is updated only `_matView` is changed.

Reply:
like this?

Mat4::multiply(scaleMat, _matView, &_tempMatView); ?

Reply:
The position has been changed, it will not update every frame now